Translation of readme in French.md

```diff
@@ -0,0 +1,51 @@
+# :wave: Passerelle vers l'Open Source | Meilleures ressources de programmation
```

Comment:
Please, This is not as Code For Community welcomes you
Have a look at the original repo's heading before the banner.
I can't help you any more with it.